### DISCUSSION TITLE
How to disable this popup ? This is kind of disturbing

### DISCUSSION THREAD

**Discussion by leninrkb** [zed-industries/zed#18377] [Feature Requests]
Every time I finish writing a single thing, it comes up
![image](https://github.com/user-attachments/assets/82ffed1d-1be3-428a-9ac5-2bf717ff717c)

**Comment by SomeoneToIgnore** [comment]
Could be done by setting this to `false`:

https://github.com/zed-industries/zed/blob/3b45bf6bb23d26a856489cb065e5e4972eaf8cce/assets/settings/default.json#L150-L152

**Reply by ghost** [reply]
![image](https://github.com/user-attachments/assets/6bb75b16-9e79-4b37-9596-135f03c67458)
Even setting to false, it still persists, it so annoying.

**Reply by SomeoneToIgnore** [reply]
This is a completion pop-up, not the signature one.
Zed has no configuration means to disable or anyhow adjust the related functionality.

**Reply by SomeoneToIgnore** [reply]
I was actually wrong, there is a setting to prevent it from appearing automatically: https://github.com/zed-industries/zed/blob/c9235ff916e4717a75d9abfc4a91df92c022f980/assets/settings/default.json#L173-L175
